tests/mem_tracker.test/runit

```diff
@@ -6,6 +6,10 @@ set -e
 dbnm=$1
 host=`cdb2sql ${CDB2_OPTIONS} -s --tabs $dbnm default 'SELECT comdb2_host()'`
 
+# Get the base address of comdb2 binary in the address space of the db process
+dbpid=`pgrep -a comdb2 | grep $dbnm | awk '{print $1}'`
```

Comment:
in clustered mode you don't have comdb2 executable running

Reply:
Right. Thank you!